policy-design-lab/pdl-frontend
# 0.15.0 Changes

This PR is for 0.15.0 release:

### Added
- Added `paymentPercentageNationwide` to Title I page map tooltips and tables [#238](https://github.com/policy-design-lab/pdl-frontend/issues/238) 

### Changed
- Changed 'Payments' to 'Benefits' for the Crop Insurance map on landing page [#240](https://github.com/policy-design-lab/pdl-frontend/issues/240)
- Changed all 'Costs' for the SNAP program to 'Benefits' across the website [#241](https://github.com/policy-design-lab/pdl-frontend/issues/241)
- Updated Title I page to use the revised API endpoints [#243](https://github.com/policy-design-lab/pdl-frontend/issues/243)

The rest of the API endpoints will be updated in 0.16.0.


### How to Test
1. Go to https://policydesignlab.ncsa.illinois.edu/title1 and compare it with https://policydesignlab.ncsa.illinois.edu/title1.
2. You should notice that besides the extra PercentageNationwide data in the map and tables, all other elements in title1 will remain as they are.
3. All map tips on the landing page and the SNAP page will use `Benefits` as the client requested in Slack.
